src/atoms/Button/Button.native.js

```diff
@@ -1,31 +1,26 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components/native';
-import { TouchableOpacity, Text } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import { TouchableOpacity } from 'react-native';
+import Text from '../Text';
 
 const StyledButton = styled(TouchableOpacity)(
   (props) => `
   background-color: ${props.theme.colors.shade[800]};
-  padding: 14px;
-  border-radius: 6px;
-  margin-top: 7px;
-`,
+  padding: 14px;`,
 );
 
-const StyledText = styled(Text)(
-  (props) => `
-  color: ${props.theme.colors.tone[900]};
-  align-self: center;
-  font-size: 18px;
-  letter-spacing: ${props.theme.fonts.letterSpacing.l};
-`,
-);
+const Button = ({ onClick, children }) => {
+  const theme = useContext(ThemeContext);
 
-const Button = ({ onClick, children, ...rest }) => (
-  <StyledButton onPress={onClick} {...rest}>
-    <StyledText {...rest}>{children}</StyledText>
-  </StyledButton>
-);
+  return (
+    <StyledButton onPress={onClick} testID="ds-button">
+      <Text color={theme.colors.tone[900]} align="center">
```

Comment:
not `tone.900`?

Reply:
It's the dummy button component, I'll just reset it to what's on master so that @shubhnik 's PR gets merged without conflict